TRUSTED-COMMITTERS.md

```diff
@@ -0,0 +1,37 @@
+# Trusted Committers
+
+Trusted committers (TCs) are those members of our working group who have elevated rights and direct write access to this repository. *TCs act as stewards of the working group and community. They aim to make consensus-based decisions in the best interest of the working group.* They also act as the guardians of this repository: TCs react to, referee, and give feedback about incoming contributions.
+
+## Current Trusted Committers
```

Comment:
we could also think about referring to a `.github/codeowners` file with the listed persons handles in it. 
That would this would give [a bit of automation](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners).

Reply:
Just do it. :)
CODEOWNERS are awesome, and it is a cheap experiment.

Reply:
I'll try that once this gets merged.